tests/basics/fun_kwargs.py

```diff
@@ -33,3 +33,15 @@ def f3(a, b, *args):
     f3(1, 2, 3, 4, a=5)
 except TypeError:
     print("TypeError")
+
+# test evaluation order of arguments
+def f4(*vargs, **kwargs):
+    print(vargs, kwargs)
+
+
+def print_ret(x):
+    print(x)
+    return x
+
+
+f4(*print_ret(["a", "b"]), kw_arg=print_ret(None))
```

Comment:
this test should go in `tests/baics/fun_kwvarargs.py`

Reply:
done